recipe/meta.yaml

```diff
@@ -11,21 +11,19 @@ source:
 
 build:
   number: {{ os.environ.get('SNOWFLAKE_SNOWPARK_PYTHON_BUILD_NUMBER', 0) }}
-  noarch: python
```

Comment:
noarch is not needed anymore because anaconda now builds two packages on linux-64 and linux-aarch64, right?

Reply:
Correct. Building it in the same arch makes the testing script easier :)